### PR TITLE
08 Testing Endpoints for Blog

### PR DESCRIPTION
**Pull request by arthurjunior250**
**What does this PR do?**
Implements Test for Blog
**Description of Task to be completed?**
Testing Blogs endpoints and checking if it returns success or error

**How should this be manually tested?**
Clone the repo

checkout to branch ft-testing-blog-endpoints-08

-Run `yarn install to install modules

-Run yarn start:dev to start

**Any background context you want to provide?**
N/A

**Screenshots (if appropriate)**
N/A

**Questions:**
- I have failed to hide nyc_output

### CODE REVIEW

File: src/routes/blog.route.js [hunk 1]
```diff
@@ -14,14 +14,9 @@ const fileFilter = (req, file, cb) => {
 };
 const uploads = multer({ storage, fileFilter });
 
-// router.post('/', checkAdmin, uploads.single("image"), saveBlog);
-// router.put('/:id', checkAdmin, updateBlog);
-// router.delete('/:id', checkAdmin, deleteBlogById);
```
Comment (PrinceNiyonshuti) [inline src/routes/blog.route.js:19]: remove the comment codes

File: src/routes/comment.route.js [hunk 2]
```diff
@@ -3,10 +3,6 @@ import { deleteComment, saveComment, getAllComments } from '../controllers/comme
 import { checkUser } from "../middleware/check";
 const router = express.Router();
 
-// router.post("/:id", checkUser, saveComment);
-// router.get("/:id", getAllComments);
-// router.delete("/:id", checkUser, deleteComment);
```
Comment (PrinceNiyonshuti) [inline src/routes/comment.route.js:8]: same applied to here

File: src/routes/newsletter.route.js [hunk 3]
```diff
@@ -3,10 +3,7 @@ import { newsLetterEmail, deleteNewsById, getAllSubscribers } from '../controlle
 import { checkAdmin } from "../middleware/check";
 const router = express.Router();
 
-// router.post('/', newsLetterEmail);
-// router.delete('/:id', checkAdmin, deleteNewsById);
```
Comment (PrinceNiyonshuti) [inline src/routes/newsletter.route.js:7]: also for here remove the unused codes

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
remove those unused codes, after everything is good to go

**Review by harerajo — APPROVED** [review]

**Pull request merged**